src/main/java/org/jenkinsci/plugins/github_branch_source/ForkPullRequestDiscoveryTrait.java

```diff
@@ -85,26 +85,34 @@
     /**
      * Constructor for stapler.
      *
-     * @param strategyId the strategy id.
-     * @param trust      the authority to use.
+     * @param strategyId
+     *            the strategy id.
+     * @param trust
+     *            the authority to use.
      */
     @DataBoundConstructor
-    public ForkPullRequestDiscoveryTrait(int strategyId,
-                                         @NonNull SCMHeadAuthority<? super GitHubSCMSourceRequest, ? extends ChangeRequestSCMHead2, ? extends SCMRevision> trust) {
+    public ForkPullRequestDiscoveryTrait(
+            int strategyId,
+            @NonNull SCMHeadAuthority<? super GitHubSCMSourceRequest, ? extends ChangeRequestSCMHead2, ? extends SCMRevision> trust) {
```

Comment:
worse than before by a long way.
see previous comment about indention based on a fixed amount vs relative.

Reply:
@jtnord 
Debatable. Do you really want to debate this?